pkg/apis/config/v1alpha1/adminqos.go

```diff
@@ -62,7 +62,12 @@ type AdminQoSConfigurationSpec struct {
 
 type AdminQoSConfig struct {
 	// ReclaimedResourceConfig is a configuration for reclaim resource
-	ReclaimedResourceConfig ReclaimedResourceConfig `json:"reclaimedResourceConfig,omitempty"`
+	// +optional
+	ReclaimedResourceConfig *ReclaimedResourceConfig `json:"reclaimedResourceConfig,omitempty"`
+
+	// EvictionConfig is a configuration for eviction
+	// +optional
+	EvictionConfig *EvictionConfig `json:"evictionConfiguration"`
```

Comment:
Would the thresholds of `drop cache` and `tune memcg` be included in this field?

Reply:
They may be included in another field such as `MemorySupressConfig` in the future, and `EvictionConfig` will only include the field that about pod eviction.